app/javascript/packages/eslint-plugin/configs/recommended.js

```diff
@@ -38,6 +38,7 @@ const config = {
     'newline-per-chained-call': 'off',
     'no-cond-assign': ['error', 'except-parens'],
     'no-console': 'error',
+    'no-continue': 'off',
```

Comment:
we only have `continue` in one spot, should we just add a per-line disable there? or do you think the rule discourages efficient code